source/session.ts

```diff
@@ -95,18 +96,20 @@ export interface ResponseError {
   error?: Data;
 }
 
-export interface MutatationOptions {
+export interface MutationOptions {
   pushToken?: string;
   additionalHeaders?: Data;
+  decodeDatesAsISO?: boolean;
```

Comment:
Personally, I usually am strict in camel-casing and don't keep acronyms capitalized. We seem to have followed this here as well, with `thumbnailUrl`,`apiEndpoint` and `strictApi`. 

Should we rename `ISO` in methods/options `Iso`?